lib/rules/no-duplicate-attr-inheritance.js

```diff
@@ -6,6 +6,43 @@
 
 const utils = require('../utils')
 
+/** @param {VElement[]} elements */
+function isConditionalGroup(elements) {
+  let hasIf = false
+  let hasElse = false
+
+  for (const element of elements) {
+    if (utils.hasDirective(element, 'if')) {
+      if (hasIf || hasElse) {
+        return false
+      }
+      hasIf = true
+    } else if (utils.hasDirective(element, 'else-if')) {
+      if (!hasIf || hasElse) {
+        return false
+      }
+    } else if (utils.hasDirective(element, 'else')) {
+      if (!hasIf || hasElse) {
+        return false
+      }
+      hasElse = true
+    } else {
+      return false
+    }
+  }
+
+  return hasIf && (elements.length === 1 || hasElse)
```

Comment:
I think this is much more readable (and we don't need to care about arrays with less than 2 entries anyway):

```suggestion
  if (elements.length < 2) {
    return false
  }

  const firstElement = element[0]
  const lastElement = elements[elements.length - 1]
  const inBetweenElements = elements.slice(1, -1)

  return (
    utils.hasDirective(firstElement, 'if') &&
    (utils.hasDirective(lastElement, 'else-if') ||
      utils.hasDirective(lastElement, 'else')) &&
    inBetweenElements.every((element) => utils.hasDirective(element, 'else-if'))
  )
```